index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Anuj Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <a href="#" id="logo">Anuj</a>
+        <nav>
+          <ul id="nav-menu">
+            <li class="nav-link"><a href="#works">Portfolio</a></li>
+            <li class="nav-link"><a href="#about-myself">About</a></li>
+            <li class="nav-link"><a href="#form-section">Contact</a></li>
+          </ul>
+          <div id="mobile-navbar">
+            <div id="hamburger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+            </div>
+            <ul id="mobile-nav-menu">
+              <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
+              <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
+              <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+            </ul>
```

Comment:
- [OPTIONAL] You don't need to have two navigation menus (mobile and desktop) you only need one that we'll customize later according to the media with the CSS properties